docs/preview/guidance/use-with-dotnet-and-functions.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using Arcus & Serilog in .NET Core and/or Azure Functions"
+layout: default
+---
+
+# Using Arcus & Serilog in .NET Core and/or Azure Functions
+
+When using Arcus Observability & Serilog it is mandatory that your application is configured correctly to ensure everything works smoothly and all features are working fine.
+
+We encourage you to follow the standard [Serilog instructions](https://github.com/serilog/serilog-aspnetcore#instructions) on setting your application up. 
+
+Some aspects we would like to highlight are:
+
+- Make sure to call [`UseSerilog`](https://www.nuget.org/packages/Serilog.AspNetCore) when creating a `IHostBuilder`
+- Remove default for logging including its configuration in `appsettings.json` *(if applicable)*
+
+If you cannot use `UseSerilog`, you can still configure it by using `AddSerilog` as a logging provider; but we recommend removing all other providers with `loggingBuilder.ClearProviders()` so that they don't interfer.
+
+## Setting up Serilog with Azure Functions
+
+Using Serilog with Azure Functions requires some guidance and we've made it a bit easier to use.
+
+Before we get started, install our NuGet package for Azure Functions:
+
+```
+PM > TODO @stijnmoreels
```

Comment:
@stijnmoreels  TODO in your PR where it's being added, not have to change now

Reply:
Yes, of course.